doc/api/errors.md

```diff
@@ -1023,6 +1023,11 @@ Used when a Node.js API is called in an unsupported manner.
 
 For example: `Buffer.write(string, encoding, offset[, length])`
 
+<a id="ERR_OPT_VALUE_OUT_OF_RANGE"></a>
+### ERR_OPT_VALUE_OUT_OF_RANGE
```

Comment:
Nit: `OPT_VALUE` is not intuitive. `ERR_VALUE_OUT_OF_RANGE` would be better I guess.